bin/webpack-helpers.js

```diff
@@ -112,6 +112,8 @@ const stableMainEntry = {
 		'./node_modules/@wordpress/components/src/checkbox-control/style.scss',
 	'spinner-style':
 		'./node_modules/@wordpress/components/src/spinner/style.scss',
+	'snackbar-notices':
+		'./node_modules/@wordpress/components/src/snackbar/style.scss',
```

Comment:
The name for this isn't following the pattern of the others (e.g. `snackbar-style`), but I don't think that matters too much.

Reply:
Doesn't matter but I favor consistency (yet somehow missed this), so thanks for pointing it out 👍